tests/e2e/test_rosacli_cluster_post.go

```diff
@@ -392,10 +392,10 @@ var _ = Describe("Healthy check",
 
 		It("with policy path will work - [id:75525]", labels.Runtime.Day1Post, labels.High,
 			func() {
-				path := profile.AccountRoleConfig.Path
-				if path == "" {
+				if profile.AccountRoleConfig == nil || profile.AccountRoleConfig.Path == "" {
```

Comment:
@radtriste why not using `path` variable by storing value `profile.AccountRoleConfig.Path` in it?
is there something which changes in the behavior when we store it into a variable and use?

Reply:
okay, got it Thanks!